docs/k0s-multi-node.md

```diff
@@ -24,13 +24,13 @@ The download script accepts the following environment variables:
 
 | Variable                    | Purpose                                           |
 |:----------------------------|:--------------------------------------------------|
-| `K0S_VERSION=v1.26.1+k0s.0` | Select the version of k0s to be installed         |
+| `K0S_VERSION=v{{{ extra.k0s_version }}}` | Select the version of k0s to be installed         |
```

Comment:
The Markdown table should be adjusted.

Reply:
I change the formatting but in fact the GH renderer doesn't care about it (I guess any markdown renderer).

Reply:
Yeah, was just a cosmetic thing for the markdown file itself.